app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -49,6 +50,10 @@ class FavoriteFragment : Fragment() {
             thisActivity.supportActionBar?.setDisplayHomeAsUpEnabled(false)
         }
 
+        rootView.findSomethingToDo.setOnClickListener {
+            findNavController(rootView).navigate(R.id.searchFragment)
```

Comment:
add slide animation. reset looks good.

Reply:
And make sure after navigating to search, the back button will navigate to events fragment.

Reply:
@liveHarshit I cannot find a function to be called so that I can add slide animation without passing a bundle, kindly help me with that .

Reply:
Use `findNavController(rootView).navigate(R.id.searchFragment, null, getAnimSlide())`

Reply:
@liveHarshit Made the changes.